packages/web/resolvers/post.ts

```diff
@@ -548,28 +538,23 @@ const PostMutations = extendType({
           data.publishedAt = new Date().toISOString()
         }
 
-        if (args.images) {
-          const headlineImage = args.images.find((i) => i.imageRole === 'HEADLINE')
-
-          if (headlineImage) {
-            await ctx.db.image.deleteMany({
-              where: {
-                postId: args.postId,
-                imageRole: 'HEADLINE',
-              },
-            })
+        if (args.headlineImage) {
+          await ctx.db.image.deleteMany({
```

Comment:
Think this can become a `delete` rather than `deleteMany`. Not particularly important though.

Reply:
Actually on second though, since the relation is on the post side, I think we can just leave the image as-is. That way if we ever want to do a sweep to remove unused images from S3 we'll still have the record. Plus it saves us a query here.

Reply:
Cool sounds good! So just to be clear, you meant let's not delete the image at all here, right? That's the change I've made but just wanna make sure we're on the same page